frigate/video.py

```diff
@@ -377,6 +378,7 @@ def receiveSignal(signalNumber, frame):
         objects_to_track,
         object_filters,
         detection_enabled,
+        improve_contrast_enabled,
```

Comment:
Rather than passing this to `process_frames`, can it be passed to the `motion_detector` instantiation above so you don't have to pass it around on every call?

Reply:
Good idea, that's actually cleaner.

Reply:
Because the motion detection is run in a subprocess and ``motion_detector`` is instantiated once, I'll just create a variable in the class that has an initial value based on the config entry. Then the call to ``detect`` doesn't have to be as messy.